src/mlpack/methods/ann/layer/elu.hpp

```diff
@@ -18,6 +19,29 @@
  * }
  * @endcode
  *
+ * Definition of the SELU function as introduced by
+ * Klambauer et. al. in Self Neural Networks.  The SELU activation
+ * function keeps the mean and variance of the input invariant.
+ *
+ * For more information, see the following paper.
+ *
+ * @code
```

Comment:
Let's put this into the class description, that way doxygen can pick this up for the documentation generation step.